documentation/guides/migrating-from-v3-to-v4.md

```diff
@@ -0,0 +1,390 @@
+# Migrating from v3 to v4
+
+Polaris v4 contains mostly internal changes and has simple migration path. This guide goes through the necessary steps to deal with breaking changes in your application.
+
+**Note:** This migration guide is for v3 but may apply for v2 as well.
+
```

Comment:
It would be good to introduce this list with a label. Something like, `Navigation components:`